src/parser/jobs/sch/modules/index.js

```diff
@@ -1,5 +1,6 @@
 import {CooldownDowntime} from './CooldownDowntime'
 import DoTs from './DoTs'
+import FaerieGauge from './Gauge'
```

Comment:
```suggestion
import {FaerieGauge} from './Gauge'
```
Project style guide - no default exports for new modules.